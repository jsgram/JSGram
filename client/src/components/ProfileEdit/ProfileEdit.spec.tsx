import { ProfileEdit } from './index';
import { shallow } from 'enzyme';
import React from 'react';

describe('Profile edit component:', () => {
    let renderer: any;
    const handleSubmit = jest.fn(() => 'some value');
    const onChangeProfile = jest.fn(() => 'some value');
    const submitting = jest.fn(() => 'some value');

    beforeEach(() => {
        renderer = shallow(<ProfileEdit
            handleSubmit={handleSubmit}
            onChangeProfile={onChangeProfile}
            submitting={submitting}
        />);
    });

    test('render - success', () => {
        expect(renderer).toMatchSnapshot();
    });
});
