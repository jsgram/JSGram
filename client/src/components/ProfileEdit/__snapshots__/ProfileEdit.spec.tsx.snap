// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile edit component: render - success 1`] = `ShallowWrapper {}`;
